client/src/css/Aaron.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap');
+
+h1,
+p,
+Table {
+  font-family: 'Inter', sans-serif;
+  margin: 0 auto;
+  text-align: center;
+}
+
+h1 {
+  padding-top: 15vh;
+  font-size: 60;
+}
```

Comment:
i think you got this from the Home.css file but you don't need these styles for your page, since you're using `className` attributes for your `<h1>` and `<p>` tags anyway